src/renderer/components/app/AppSettings.tsx

```diff
@@ -180,6 +182,7 @@ export const AppSettings: React.FC<Props> = (props): JSX.Element => {
   return (
     <Styled.Container>
       <Styled.Title>{intl.formatMessage({ id: 'setting.app.title' })}</Styled.Title>
+      <AccountAddressSelector addresses={[{ walletAddress: 'bnb123123', walletType: 'ledger', chain: BNBChain }]} />
```

Comment:
Just in case you want to see `AccountAddressSelector` in right place, you can put into `WalletPoolActionsHistoryHeader` here: 
https://github.com/thorchain/asgardex-electron/blob/93baaf3f02e4c4a21fbe32d7e21c84365a2050f9/src/renderer/components/poolActionsHistory/WalletPoolActionsHistoryHeader.tsx#L29